pkg/fakerp/manifest.go

```diff
@@ -39,7 +39,7 @@ func LoadClusterConfigFromManifest(log *logrus.Entry, manifestTemplate string, c
 		manifestTemplate = conf.Manifest
 	}
 	if manifestTemplate == "" {
-		manifestTemplate = "test/manifests/normal/create.yaml"
+		manifestTemplate = "../../test/manifests/normal/create.yaml"
```

Comment:
The preceeding `../..` are for my own debug purposes.  Will need to reset this pathing before merge.